### DISCUSSION TITLE
Add Apple Provider - Social Authentication

### DISCUSSION THREAD

**Discussion by darshithedpara** [adonisjs/core#1250] [Ideas]
- Facebook provider
- Twitter provider
- Google provider
- Github provider
- LinkedIn provider

We must add **Apple Provider** alogn with above standard providers.

**Comment by RomainLanz** [comment]
It will be added when we migrate the [`Ally` module](https://github.com/adonisjs/adonis-ally) to V5.

**Reply by DraftProducts** [reply]
Hi @RomainLanz ! 
Do you have any idea when the migration of this functionality is planned?

**Reply by DraftProducts** [reply]
Sorry for the notification, i found my answer on this trello card:
https://trello.com/c/AF42y3JN/11-social-authentication

**Comment by fredmaiaarantes** [comment]
It says Q4 now but it has been postponed before, do you think are you able to make it in Q4? @RomainLanz 
I'm looking for alternatives but didn't want to use something outside AdonisJS. :/

**Reply by thetutlage** [reply]
To be honest, we are a little late on some of the important features. I will kick starting development in couple of days and hopefully we can get this done within this month.

**Reply by soonzhi** [reply]
any alternative to social auth?

**Comment by CodingDive** [comment]
Are there any additional ally providers for v6 by the community? Microsoft / Apple?
